lerf/encoders/openclip_encoder.py

```diff
@@ -67,8 +73,8 @@ def name(self) -> str:
     def embedding_dim(self) -> int:
         return self.config.clip_n_dims
 
-    def set_positives(self, text_list):
-        self.positives = text_list
+    def set_positives(self, handle):
```

Comment:
Two questions (changing `set_positives`'s args to `text_list` -> `handle`):
1.  Is `handle` ever used in `set_positives`? Is this variable necessary?
2. Does this remove the ability to feed in a text prompt directly (via code w/o viewer)? Or is there a way to do this via viser? My main worry is that this code needs to be refactored for `ns-render` later on.

Reply:
true yea should change it to be more modular